tests/ecc_test.c

```diff
@@ -520,6 +520,8 @@ static int _ecc_new_api(void)
       DO(ecc_verify_hash(buf, len, data16, 16, &stat, &pubkey));
       if (stat != 1) return CRYPT_FAIL_TESTVECTOR;
 
+#ifndef USE_TFM
```

Comment:
better to check at runtime `if (strcmp(ltc_mp.name, "TomsFastMath") != 0)` as it's possible to build for all MPI providers, with `USE_TFM`, but then select another one when invoking [`./test`](https://github.com/libtom/libtomcrypt/blob/develop/tests/test.c#L341)

Reply:
OK. Should I change the existing test for reading compressed points as well? That'll have the same issue with switching MPI provider.